charts/pomerium/README.md

```diff
@@ -325,6 +325,7 @@ A full listing of Pomerium's configuration variables can be found on the [config
 | `ingress.hosts`                                              | Ingress accepted hostnames                                                                                                                                                                                                                                                                         | `[]`                                                                        |
 | `ingress.secretName`                                         | Ingress TLS certificate secret name                                                                                                                                                                                                                                                                | `[]`                                                                        |
 | `ingress.tls.hosts`                                          | Override automatic ingress tls hosts list                                                                                                                                                                                                                                                          | `[]`                                                                        |
+| `ingress.authenticate.subdomain`                             | Setting a custom authenticate url by setting a subdomain                                                                                                                                                                                                                                           | `authenticate`                                                              |
```

Comment:
I think the parameter for this should be `name` rather than `subdomain`.  So `ingress.authenticate.name`, etc.

Reply:
ok done